web-client/scripts/build-web.sh

```diff
@@ -0,0 +1,8 @@
+wasm-pack build --weak-refs --target web --out-name index --out-dir dist/web/main-wasm &&
```

Comment:
I'd suggest to use `set -e` at the beginning instead of `&&` for each line.

Reply:
Fixed by you in 67d08d40e67bfcfca7aa79de98b34a5891bc9271 already :+1: